portfolio/src/main/webapp/index.html

```diff
@@ -1,14 +1,24 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+
+  <body bgcolor="#ff9999">
+    <h1 class="title">Megha Kak</h1>
+    <div class="top-navigation"> 
+        <a href="index.html">Home</a>
+        <a href="hobbies.html">Hobbies</a>
+        <a href="gallery.html">Gallery</a>
+        <a href="blog.html">STEP Blog</a>
```

Comment:
nit: Indent by 2 spaces at a time. Don’t use tabs or mix tabs and spaces for indentation.
https://google.github.io/styleguide/htmlcssguide.html#Indentation

I think these are indented by 4 spaces, please change to 2 to be consistent with rest of the file